api/stream/design.go

```diff
@@ -224,6 +224,20 @@ var _ = Service("stream", func() {
 		})
 	})
 
+	Method("ListDeletedSources", func() {
```

Comment:
Did not we want just same GET endpoint as `listSources` but filter on `Param(isDeleted)` level? So if `isDeleted` is provided, it filters out sources based on deleted flag, otherwise is lists regularly.

Reply:
No, for deleted separate endpoint is okay in my opinion. I'd use filters for enabled / disabled.